babel.config.js

```diff
@@ -0,0 +1,16 @@
+const presets = [
```

Comment:
does this work to share babel across all the folders?

Reply:
I'm not sure, my editor was complaining that I didn't have a babel config at the root of the project though. I added it thinking it wouldn't impact anything, do you think it will?